pyrobosim/pyrobosim/data/pddlstream/domains/05_nav_grasp_stream/streams.pddl

```diff
@@ -0,0 +1,65 @@
+; STREAMS FOR PDDL PLANNING DOMAIN (NAVIGATION + MANIPULATION + GRASP STREAMS)
+;
+; Contains cost functions for actions and streams for navigation and manipulation.
+;
+; Accompanying planning domain defined in the `domain.pddl` file.
+
+(define (stream stream_nav_stream)
```

Comment:
Judging by the folder name this should be:

```suggestion
(define (stream nav_grasp_stream)
```

Reply:
yes, thank you!